src/app/organizations/settings/subscription-hidden.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from "@angular/core";
+
+// Component is used so that the SVG can embed CSS color variables
+@Component({
+  selector: "app-org-subscription-hidden",
```

Comment:
Thoughts on standardizing some form of naming convention for svg? It's a bit unclear looking at the parent component that this is only an icon.

Reply:
`app-icon-subcription-hidden` maybe?

Reply:
Good idea, but I'd prefer `image` as the more generic descriptor.

So that would be
`app-image-org-subscription-hidden` (selector)
`ImageSubscriptionHiddenComponent` (class name)
`image-subscription-hidden.component` (file name)

?

Not sure if there's a clear advantage between prefix and suffix, prefix has the advantage of collecting them together in the folder structure.

Reply:
🤷 either works. I mostly want to easily identify that it's an image.

Reply:
This is done